src/test/resources/org/rust/ide/folding/fixtures/custom_region.rs

```diff
@@ -0,0 +1,4 @@
+<fold text='custom text'>//region custom text
+//some code
```

Comment:
Maybe add some real code here instead of comment?

Reply:
Yeah, I'm working on it right now

Reply:
I run into some problems here